pkgs/development/node-packages/default-v10.nix

```diff
@@ -29,6 +29,22 @@ nodePackages // {
     buildInputs = [ nodePackages.node-gyp-build pkgs.libtool pkgs.autoconf pkgs.automake ];
   };
 
+  dat-fox-helper = nodePackages."dat-fox-helper-git://github.com/sammacbeth/dat-fox-helper#v0.2.1 ".override {
```

Comment:
I am not sure if this flake like syntax is allowed in nixpkgs.

Reply:
And yet it's necessary, as the package is not present on npm and so it has to be referenced by url